### PR TITLE
Admixer ID System: add userId submodule 

### PR DESCRIPTION
**Pull request by Galphimbl**
## Type of change

- [x] Feature
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change

Adds AdmixerId System to the user id module

prebid.github.io PR [#2666](https://github.com/prebid/prebid.github.io/pull/2666)

### CODE REVIEW

**Comment by jdwieland8282** [comment]
Docs PR https://github.com/prebid/prebid.github.io/pull/2666

**Comment by smenzer** [comment]
@jlukas79 have you had a chance to review this yet?

**Review by patmmccann — APPROVED** [review]
Code mostly looks good, only one change I can see:

File: modules/userId/eids.js [hunk 1]
```diff
@@ -172,6 +172,12 @@ const USER_IDS_CONFIG = {
   'tapadId': {
     source: 'tapad.com',
     atype: 1
+  },
+
+  // Admixer Id
+  '__adm__admixer': {
+    source: 'admixer.net',
+    atype: 1
```
Comment (patmmccann) [inline modules/userId/eids.js:180]: aren't you agent type 3?

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by patmmccann** [comment]
@Galphimbl can you also resolve conflicts?

File: modules/admixerIdSystem.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+   */
+  gvlid: 511,
+  /**
+   * decode the stored id value for passing to bid requests
+   * @function
+   * @param {string} value
+   * @returns {{__adm__admixer:string}}
+   */
+  decode(value) {
+    return { '__adm__admixer': value }
```
Comment (smenzer) [inline modules/admixerIdSystem.js:34]: This is the value that gets exposed to bidder adapters...to follow standards it should be `admixerId`

File: modules/userId/eids.js [hunk 3]
```diff
@@ -172,6 +172,12 @@ const USER_IDS_CONFIG = {
   'tapadId': {
     source: 'tapad.com',
     atype: 1
+  },
+
+  // Admixer Id
+  '__adm__admixer': {
```
Comment (smenzer) [inline modules/userId/eids.js:178]: Following convention this should be `admixerId`

**Review by smenzer — CHANGES_REQUESTED** [review]
I still need to do a deeper review but these were two obvious things for me

**Review by smenzer — CHANGES_REQUESTED** [review]
finished doing a full review...code looks good with the minor comments I made.

you should also add tests to `userId_spec.js` to ensure full coverage.

**Comment by Galphimbl** [comment]
@smenzer @patmmccann upd

**Comment by smenzer** [comment]
@Galphimbl changes look good. I would suggest doing a rebase on master since it looks like you've pulled in some other commits and I just want to make sure this is a clean PR (I see some "new" code from UID and Novatiq, for example, which is already in the PBJS master).

**Comment by Galphimbl** [comment]
@smenzer i've just merge master into PR and resolve conflicts.

**Review by patmmccann — APPROVED** [review]

**Review by smenzer — APPROVED** [review]
LGTM

**Pull request merged**